### PR TITLE
ftr: add stop watching configuration docs

### PR DESCRIPTION
**Pull request by LaurenceLiZhixin**
Thank you for helping make the Dapr documentation better!

**Please follow this checklist before submitting:**

- [x] [Read the contribution guide](https://docs.dapr.io/contributing/contributing-docs/)
- [x] Commands include options for Linux, MacOS, and Windows within codetabs
- [x] New file and folder names are globally unique
- [x] Page references use shortcodes instead of markdown or URL links
- [x] Images use HTML style and have alternative text
- [x] Places where multiple code/command options are given have codetabs

In addition, please fill out the following to help reviewers understand this pull request:

### CODE REVIEW

File: daprdocs/content/en/developing-applications/building-blocks/configuration/howto-manage-configuration.md [hunk 1]
```diff
@@ -108,5 +108,25 @@ message SubscribeConfigurationRequest {
 
 Using this method, you can subscribe to changes in specific keys for a given configuration store. gRPC streaming varies widely based on language - see the [gRPC examples here](https://grpc.io/docs/languages/) for usage.
 
+### Stop watching configuration items
+
+After you invoked the Watch Configuration Items API mentioned above, the gRPC-Server Stream would start. This thread is not closed by itself, and you can do by explicitly calling the Unsubscribe API. The method accepts the following request object:
```
Comment (msfussell) [inline daprdocs/content/en/developing-applications/building-blocks/configuration/howto-manage-configuration.md]:
```suggestion
After you have subscribed to watch configuration items, the gRPC-server stream starts. This stream thread does not close itself, and you have to do by explicitly calling the `UnSubscribeConfigurationRequest` API. This method accepts the following request object:
```

File: daprdocs/content/en/developing-applications/building-blocks/configuration/howto-manage-configuration.md [hunk 2]
```diff
@@ -108,5 +108,25 @@ message SubscribeConfigurationRequest {
...
+
+```proto
+// UnSubscribeConfigurationRequest is the message to stop watching the key-value configuration.
+message UnSubscribeConfigurationRequest {
+  // The name of configuration store.
+  string store_name = 1;
+
+  // Optional. The keys of the configuration item to stop watching.
+  // Store_name and keys should match previous SubscribeConfigurationRequest's keys and store_name.
+  // Once invoked, the previous subscription that watching update event would return.
```
Comment (msfussell) [inline daprdocs/content/en/developing-applications/building-blocks/configuration/howto-manage-configuration.md]:
```suggestion
  // Once invoked, the subscription that is watching the update for the key-value event is stopped
```

File: daprdocs/content/en/developing-applications/building-blocks/configuration/howto-manage-configuration.md [hunk 3]
```diff
@@ -108,5 +108,25 @@ message SubscribeConfigurationRequest {
...
+  repeated string keys = 2;
+}
+```
+
+Using this method, you can stop watching the update event of configuration. Dapr locates the subscription stream that will be closed based on store_name and keys, and closes it.
```
Comment (msfussell) [inline daprdocs/content/en/developing-applications/building-blocks/configuration/howto-manage-configuration.md]:
```suggestion
Using this unsubscribe method, you can stop watching configuration update events. Dapr locates the subscription stream based on the `store_name` and any optional keys supplied, and closes it.
```

Comment (msfussell) [reply daprdocs/content/en/developing-applications/building-blocks/configuration/howto-manage-configuration.md]: @LaurenceLiZhixin  - Please review my suggestions.

**Comment by yaron2** [comment]
ping @LaurenceLiZhixin

File: daprdocs/content/en/developing-applications/building-blocks/configuration/howto-manage-configuration.md (hunk 3 above)
Comment (LaurenceLiZhixin) [reply daprdocs/content/en/developing-applications/building-blocks/configuration/howto-manage-configuration.md]: Thanks for your suggestions, and I changed my sentences.@msfussell

**Comment by msfussell** [comment]
ping @LaurenceLiZhixin

**Comment by greenie-msft** [comment]
@LaurenceLiZhixin - Did you intend to also change the docsy theme as part of this commit? I believe this commit includes changes to  daprdocs/themes/docsy.

**Comment by msfussell** [comment]
@LaurenceLiZhixin  - Pinging. Would like to get this change in for v1.6 release

**Comment by LaurenceLiZhixin** [comment]
I think it's fine to get this change in v1.6, I'm not familiar with the front end. What's the need of the change of theme?

**Comment by LaurenceLiZhixin** [comment]
The continue work of this pr is moved to https://github.com/dapr/dapr/pull/4130#partial-pull-merging , as this pr is containing many conflicts when i fix dco.

**Comment by greenie-msft** [comment]
@LaurenceLiZhixin, This contribution should not include any changes to the docsy theme. Can you please remove those changes? You can see all the changes when you look at "files changed".

**Comment by greenie-msft** [comment]
@LaurenceLiZhixin - can you please see my latest comment regarding removal of the docsy changes. Only checkin this file: /howto-manage-configuration.md

**Comment by greenie-msft** [comment]
PR addressing this merged: https://github.com/dapr/docs/pull/2147

**Pull request closed without merging**